include/sampling/sampling.hpp

```diff
@@ -239,65 +219,40 @@ void logconcave_sampling(PointList &randPoints,
                          const Point &starting_point,
                          unsigned int const& nburns,
                          NegativeGradientFunctor &F,
-                         NegativeLogprobFunctor &f)
-{
-    typedef typename WalkTypePolicy::template Walk
-            <
-                    Point,
-                    Polytope,
-                    RandomNumberGenerator,
-                    NegativeGradientFunctor,
-                    NegativeLogprobFunctor,
-                    Solver
-            > walk;
-
-    typedef typename WalkTypePolicy::template parameters
-            <
-                    NT,
-                    NegativeGradientFunctor
-            > walk_params;
-
-    // Initialize random walk parameters
```

Comment:
there is probably some mistake deleting this

Reply:
I do not use walk_params anymore.